pkg/cmd/factory/factory_impl.go

```diff
@@ -90,6 +90,12 @@ func (f *factoryImpl) GetCredential() *security.Credential {
 
 // EtcdClient creates new cdc etcd client.
 func (f *factoryImpl) EtcdClient() (*etcd.CDCEtcdClient, error) {
```

Comment:
Why putting this check here? Seems confusing why getting an `etcd.CDCEtcdClient` needs to check for PD address's validity.

Reply:
It can moved to other places. The reason I put it here is that, for now, the etcd address(es) should be and could only be the addresses of a pd cluster.